portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,20 +36,33 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+
     
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
+  private final ArrayList<String> comments = new ArrayList<>();
+
```

Comment:
nit: Same comment here - one newline should be sufficient

Reply:
Done.